tests/unit/test_score_matching.py

```diff
@@ -793,10 +776,10 @@ def log_pdf(y_: ArrayLike) -> ArrayLike:
         true_score_result = true_score(x)
 
         # Define a sliced score matching object
-        score_key, _ = random.split(self.random_key)
+        score_key, _ = jr.split(self.random_key)
         sliced_score_matcher = coreax.score_matching.SlicedScoreMatching(
             score_key,
-            random_generator=random.rademacher,
+            random_generator=jr.rademacher,
             use_analytic=True,
             num_epochs=50,
```

Comment:
This test takes _ages_ currently - consider changing this to make it faster!
```suggestion
            num_epochs=5,
```
Also consider a similar change for `test_univariate_gmm_score` - although that one currently needs at least ~30 epochs to still pass with its current sensitivity of 0.5, so may need its sensitivity value lowering to allow a lower number of epochs so that the test can run in a reasonable amount of time.

Reply:
(I realise this may be out of scope for this PR though - if so, I can create a separate issue for this.)

Reply:
Made these changes!